templates/orbital-sync.xml

```diff
@@ -0,0 +1,59 @@
+    <Config Name="Secondary Pi-hole 5 password" Target="SECONDARY_HOST_5_PASSWORD" Default="" Mode="" Description="The password used to log in to the admin interface." Type="Variable" Display="advanced" Required="false" Mask="true" />
+    <Config Name="Secondary Pi-hole 6 base url" Target="SECONDARY_HOST_6_BASE_URL" Default="" Mode="" Description="The base URL of your secondary Pi-hole, including the scheme (HTTP or HTTPS) and port but not including a following slash. i.e. 'http://192.168.1.8' or 'https://pihole7.example.com'" Type="Variable" Display="advanced" Required="false" Mask="false" />
+    <Config Name="Secondary Pi-hole 6 password" Target="SECONDARY_HOST_6_PASSWORD" Default="" Mode="" Description="The password used to log in to the admin interface." Type="Variable" Display="advanced" Required="false" Mask="true" />
+    <Config Name="Secondary Pi-hole 7 base url" Target="SECONDARY_HOST_7_BASE_URL" Default="" Mode="" Description="The base URL of your secondary Pi-hole, including the scheme (HTTP or HTTPS) and port but not including a following slash. i.e. 'http://192.168.1.9' or 'https://pihole8.example.com'" Type="Variable" Display="advanced" Required="false" Mask="false" />
+    <Config Name="Secondary Pi-hole 7 password" Target="SECONDARY_HOST_7_PASSWORD" Default="" Mode="" Description="The password used to log in to the admin interface." Type="Variable" Display="advanced" Required="false" Mask="true" />
+    <Config Name="Secondary Pi-hole 8 base url" Target="SECONDARY_HOST_8_BASE_URL" Default="" Mode="" Description="The base URL of your secondary Pi-hole, including the scheme (HTTP or HTTPS) and port but not including a following slash. i.e. 'http://192.168.1.10' or 'https://pihole9.example.com'" Type="Variable" Display="advanced" Required="false" Mask="false" />
+    <Config Name="Secondary Pi-hole 8 password" Target="SECONDARY_HOST_8_PASSWORD" Default="" Mode="" Description="The password used to log in to the admin interface." Type="Variable" Display="advanced" Required="false" Mask="true" />
+    <Config Name="Secondary Pi-hole 9 base url" Target="SECONDARY_HOST_9_BASE_URL" Default="" Mode="" Description="The base URL of your secondary Pi-hole, including the scheme (HTTP or HTTPS) and port but not including a following slash. i.e. 'http://192.168.1.11' or 'https://pihole10.example.com'" Type="Variable" Display="advanced" Required="false" Mask="false" />
+    <Config Name="Secondary Pi-hole 9 password" Target="SECONDARY_HOST_9_PASSWORD" Default="" Mode="" Description="The password used to log in to the admin interface." Type="Variable" Display="advanced" Required="false" Mask="true" />
```

Comment:
These are redundant, having some text in the overview on how to add more than 2 would be prefered.